types/semver-sort/index.d.ts

```diff
@@ -1,7 +1,12 @@
 // Type definitions for semver-sort 0.0
 // Project: https://github.com/ragingwind/semver-sort
-// Definitions by: Leonid Logvinov <https://github.com/LogvinovLeon>
+// Definitions by: Leonid Logvinov <https://github.com/LogvinovLeon>, Dániel Tar <https://github.com/qcz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare function desc(versions: string[]): string[];
-export = desc;
+declare function asc(versions: ReadonlyArray<string>): string[];
+declare function desc(versions: ReadonlyArray<string>): string[];
+
+export {
+	asc,
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/semver-sort/index.d.ts;
  Line: 10;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.